datalad/config.py

```diff
@@ -81,7 +82,10 @@ def _parse_gitconfig_dump(dump, store, fileset, replace):
             continue
         if line.startswith('file:'):
             # origin line
-            fileset.add(abspath(line[5:]))
+            fname = line[5:]
+            if not isabs(fname):
+                fname = opj(cwd, fname) if cwd else abspath(fname)
+            fileset.add(fname)
```

Comment:
Fix looks good.

As far as I can see, the main consequence of the faulty expansion here is that the mtime check with `.reload(force=False)` could be wrong.

Reply:
Correct.